contrib/fparser/fparser_ad.cc

```diff
@@ -843,6 +877,17 @@ bool FunctionParserADBase<Value_t>::JITCompileHelper(const std::string & Value_t
           // --sp; ccfile << "s[" << sp << "] = s[" << sp << "] < s[" << (sp+1) << "] ? std::log(s[" << (sp+1) << "]) - 1.5 + 2.0/s[" << (sp+1) << "] * s[" << sp << "] - 0.5/(s[" << (sp+1) << "]*s[" << (sp+1) << "]) * s[" << sp << "]*s[" << sp << "] : std::log(s[" << sp << "]);\n";
           --sp; ccfile << "s[" << sp << "] = s[" << sp << "] < s[" << (sp+1) << "] ? std::log(s[" << (sp+1) << "])  +  (s[" << sp << "]-s[" << (sp+1) << "])/s[" << (sp+1) << "]  -  std::pow((s[" << sp << "]-s[" << (sp+1) << "])/s[" << (sp+1) << "],2.0)/2.0  +  std::pow((s[" << sp << "]-s[" << (sp+1) << "])/s[" << (sp+1) << "],3.0)/3.0 : std::log(s[" << sp << "]);\n";
         }
+        else if (function == mFErf)
+        {
+#if LIBMESH_HAVE_CXX11_ERF
```

Comment:
This is the JIT compiled stuff though, and it *is* currently using `std::erf` (per line below). In order to use the Faddeeva methods in the JIT compiled file we would need to add the include of the Faddeeva header and make sure the JIT compiler also compiles the Faddeeva source, which I suppose is not hard.